ci-operator/step-registry/rosa/operator/cleanup/rosa-operator-cleanup-commands.sh

```
@@ -28,6 +42,38 @@ log "Cleaning up test operator resources"
+        fi
+        sleep 5
+    done
+fi
+
+# Clear CRD ownerReferences left by the e2e ClusterPackage so the
+# production ClusterPackage can re-adopt them. Without this, PKO
+# refuses adoption with "not owned by previous revision".
+# Only clear if the CRD's instance label matches our e2e package name
+# AND differs from the production operator name.
+if [[ -n "${OPERATOR_CRDS:-}" ]]; then
+    OPERATOR_NAME_BASE=$(echo "${CLUSTER_PACKAGE_NAME}" | sed 's/-e2e-test$//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)